Integrations/integration-CyberTriage.yml

```diff
@@ -180,25 +182,31 @@ script:
         is_true = lambda x: x == 'yes'
         is_hash_upload_on = is_true(CMD_ARGS['malware_hash_upload']) # arg value = 'yes' or 'no'
         is_file_upload_on = is_true(CMD_ARGS['malware_file_upload']) # arg value = 'yes' or 'no'
-        is_fast_scan_on = not(is_true(CMD_ARGS['full_scan']))        # arg value = 'yes' or 'no'
         endpoint = CMD_ARGS['endpoint']
+        scan_options = CMD_ARGS['scan_options']
+        incident_name = CMD_ARGS.get('incident_name','Default') # If no incident name is provided it is put into the "Default" incident
```

Comment:
i think its better to give the argument the default value `Default` instead in the code, the use will see that this is the default
please add whitespace after the comma